server/index.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+
+app.use(express.static('./public'));
```

Comment:
You're in the server folder here. Relative path to public would be '../public'. Unless you plan on moving the folder.